spec/pom.xml

```diff
@@ -16,24 +16,38 @@
     SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
 
 -->
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
     <parent>
         <groupId>org.eclipse.ee4j.jms</groupId>
         <artifactId>jms-parent</artifactId>
         <version>3.0-SNAPSHOT</version>
```

Comment:
Fix parent version.

Reply:
I put the spec and main pom at 3.0-SNAPSHOT here, since the spec uses x.y. The api/ module doesn't use the parent and is at 3.0.0-SNAPSHOT.

Would you like to have them all at 3.0.0-SNAPSHOT?